core/src/androidMain/kotlin/ApplicationContextInitializer.kt

```diff
@@ -0,0 +1,15 @@
+package com.juul.kable
+
+import android.content.Context
+import androidx.startup.Initializer
+
+internal lateinit var applicationContext: Context
+
+public class ApplicationContextInitializer : Initializer<Unit> {
```

Comment:
Each `T` of `Initializer<T>` can only have one initializer, so providing it like this prevents library consumers from having their own `Initializer<Unit>` -- I feel like we shouldn't be consuming such a fundamental type.

I think my preferred option here is a just adding a marker `object Kable`, and just making this `public class KableInitializer : Initializer<Kable>`. As is, the public class name is kind of exposing the implementation detail.

Reply:
Good callout and great suggestion!